data/changelog/ppxlib/2023-06-20-ppxlib-0.30.0.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Ppxlib 0.30.0
+date: "2023-06-20"
+tags: [ppxlib, release]
+changelog: |
+  * Adopt the OCaml Code of Conduct on the repo ([ocaml-ppx/ppxlib#426](https://github.com/ocaml-ppx/ppxlib/pull/426), @pitag-ha)
+  * Clean up misleading attribute hints when declared for proper context. ([ocaml-ppx/ppxlib#425](https://github.com/ocaml-ppx/ppxlib/pull/425), @ceastlund)
+  * Ast_pattern now has ebool, pbool helper, and a new map.([ocaml-ppx/ppxlib#402](https://github.com/ocaml-ppx/ppxlib/pull/402), @Burnleydev1)
+  * multiple errors are now reported in `metaquot`. ([ocaml-ppx/ppxlib#397](https://github.com/ocaml-ppx/ppxlib/pull/397), @Burnleydev1)
+  * Add `Attribute.declare_with_attr_loc` ([ocaml-ppx/ppxlib#396](https://github.com/ocaml-ppx/ppxlib/pull/396), @dvulakh)
+  * Add "ns" and "res" as reserved namespaces([ocaml-ppx/ppxlib#388](https://github.com/ocaml-ppx/ppxlib/pull/388), @davesnx)
+  * Make quoter `let` binding non-recursive ([ocaml-ppx/ppxlib#401](https://github.com/ocaml-ppx/ppxlib/pull/401), @sim642)
```

Comment:
```suggestion
  * Make Quoter `let` binding non-recursive ([ocaml-ppx/ppxlib#401](https://github.com/ocaml-ppx/ppxlib/pull/401), @sim642)
```
Capital Q if this is a specific company or tool. Is it the same as [this](https://www.quoter.com/)?

Reply:
I'd say "quoter" is a workflow that helps write a hygienic PPX and "`Quoter`" is the module inside Ppxlib meant to implement that workflow. 
```suggestion
  * Make `Quoter`'s `let` binding non-recursive ([ocaml-ppx/ppxlib#401](https://github.com/ocaml-ppx/ppxlib/pull/401), @sim642)
```
Btw, will the `\`` be rendered correctly in the changelog?